uxarray/grid.py

```diff
@@ -507,7 +511,6 @@ def _build_edge_node_connectivity(self):
             if inverse_indices[i] != INT_FILL_VALUE:
                 inverse_indices[i] -= indexes[i]
 
-        # add Mesh2_edge_nodes to internal dataset
         self.ds['Mesh2_edge_nodes'] = xr.DataArray(
```

Comment:
If `Mesh2_edge_nodes` is already in our dataset, we probably have to rewrite this initialization right here, since a DataArray already exists for it

Reply:
If `Mesh2_edge_nodes` is already in our dataset but the `inverse_indices` doesn't exist, what we do is we will force-repopulate the `Grid.edge_nodes_connectivity`  meaning that we're going to rewrite things in our own way

Reply:
It appears that when `repopulate` is true, it computes derives the `edge_nodes` using our approach, and when it is false, it only does the inverse indices. Is this the behavior that we want?